src/allotropy/allotrope/schema_parser/path_util.py

```diff
@@ -67,9 +81,10 @@ def get_schema_path_from_reference(reference: str) -> Path:
     return Path(f"{ref_match.groups()[0]}.json")
 
 
-def get_model_file_from_schema_path(schema_path: Path) -> Path:
+def get_model_path_from_schema_path(schema_path: Path) -> Path:
     rel_schema_path = PureWindowsPath(get_rel_schema_path(schema_path))
     schema_file = rel_schema_path.name
+    # TODO: remove lower?
```

Comment:
why/why not?

Reply:
Oh oops, I leave these to follow up myself before putting up PR 👀 . For the record - it was removed because it wasn't needed for the model filename, which is never capitalized for either the model or the schema

It is needed below this line for the `REC` -> `rec` transform when converting a schema path to a model path